bin/flume-ng

```diff
@@ -222,7 +226,8 @@ run_flume() {
 # set default params
 FLUME_CLASSPATH=""
 FLUME_JAVA_LIBRARY_PATH=""
-JAVA_OPTS="-Xmx20m"
+JAVA_OPTS=""
+JAVA_HEAP_MAX ="-Xmx20m"
```

Comment:
Please remove the extra space before the `=`, it results in runtime error (`bin/flume-ng: line 230: JAVA_HEAP_MAX: command not found`). Thank you.